python/llm/example/cpp-python/README.md

```diff
@@ -0,0 +1,28 @@
+# BigDL-LLM INT4 Inference Using Llama-Cpp-Python Format API
+
+In this example, we show how to run inference on converted INT4 model using llama-cpp-python format API.
```

Comment:
> Add reference to our docs to this API. https://github.com/orgs/analytics-zoo/projects/21/views/8?pane=issue&itemId=32407096. Use the same name as our docs.

But I think we don't have llm document yet. The links (https://github.com/intel-analytics/BigDL/tree/main/python/llm#install and https://github.com/intel-analytics/BigDL/tree/main/python/llm#convert-model) are temporarily used in `python/llm/example/cpp-python/README.md` and `python/llm/example/langchain/README.md`, we can update it when our document is ready.

Reply:
I think we may start to auto generate our API doc first? it should not be a hard task and we could just put it under nano's API doc tree. This could help us keep a (maybe not easy to read, but at lease up-to-date) API reference. (don't do this in this PR)